### PR TITLE
docs: add https://github.com/VictoriaMetrics/ansible-playbooks to README.md

### PR DESCRIPTION
**Pull request by rodrigc**
Signed-off-by: Craig Rodrigues <rodrigc@crodrigues.org>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1834,8 +1834,9 @@ It is safe sharing the collected profiles from security point of view, since the
   See [these docs](https://github.com/netdata/netdata#integrations).
 * [go-graphite/carbonapi](https://github.com/go-graphite/carbonapi) can use VictoriaMetrics as time series backend.
   See [this example](https://github.com/go-graphite/carbonapi/blob/main/cmd/carbonapi/carbonapi.example.victoriametrics.yaml).
-* [Ansible role for installing single-node VictoriaMetrics](https://github.com/dreamteam-gg/ansible-victoriametrics-role).
-* [Ansible role for installing cluster VictoriaMetrics](https://github.com/Slapper/ansible-victoriametrics-cluster-role).
+* [Ansible role for installing cluster VictoriaMetrics (VictoriaMetrics provided)](https://github.com/VictoriaMetrics/ansible-playbooks).
+* [Ansible role for installing single-node VictoriaMetrics (community provided)](https://github.com/dreamteam-gg/ansible-victoriametrics-role).
+* [Ansible role for installing cluster VictoriaMetrics (community provided)](https://github.com/Slapper/ansible-victoriametrics-cluster-role).
```
Comment (hagen1778) [inline README.md:1839]:
```suggestion
* [Ansible role for installing cluster VictoriaMetrics (by VictoriaMetrics)](https://github.com/VictoriaMetrics/ansible-playbooks).
* [Ansible role for installing cluster VictoriaMetrics (by community)](https://github.com/Slapper/ansible-victoriametrics-cluster-role).
* [Ansible role for installing single-node VictoriaMetrics (by community)](https://github.com/dreamteam-gg/ansible-victoriametrics-role).

```

**Review by hagen1778 — CHANGES_REQUESTED** [review]
Thanks! Please see the suggestion.

Please also run `make docs-sync` command from the project root to compile the docs.

**Comment by rodrigc** [comment]
> Thanks! Please see the suggestion.
> 
> Please also run `make docs-sync` command from the project root to compile the docs.

@hagen1778 OK, I added your suggestion and ran `make docs-sync`

**Review by hagen1778 — APPROVED** [review]
Thank you!

**Pull request merged**